assets/js/script.js

```diff
@@ -1,12 +1,66 @@
-document.getElementById('add-button').addEventListener('click', function() {
+document.getElementById('add-button').addEventListener('click', function () {
     const input = document.getElementById('item-input');
     const listContainer = document.getElementById('list-container');
     const listItem = document.createElement('li');
 
     if (input.value.trim() !== '') {
         listItem.innerHTML = `${input.value} <button class="delete is-small"> </button>`;
         listContainer.appendChild(listItem);
-        input.value = ''; 
+        input.value = '';
 
     }
 });
+
+// Modal JS Functionality Implementation Below:
+
+// Clicking the 'x' button in the top right: 
+document.addEventListener('DOMContentLoaded', function () {
+    const modal = document.querySelector('.modal');
+    const modalButton = document.querySelector('.button');
+    const closeButton = modal.querySelector('.delete');
+
+    // Fx to open the modal
+    function openModal() {
+        modal.classList.add('is-active');
+    }
+
+    // Fx to close the modal
+    function closeModal() {
+        modal.classList.remove('is-active');
+    }
+
+    // Event listener for the button to open the modal
+    modalButton.addEventListener('click', closeModal);
+
+    // Event listener for the close button within the modal
+    closeButton.addEventListener('click', closeModal);
+});
+
+
+
+// Clicking the 'Let's Get Started!' button:
+document.addEventListener('DOMContentLoaded', function () {
+    const modal = document.querySelector('.modal');
+    const modalButton = document.querySelector('.button');
+    const closeButton = modal.querySelector('.delete');
+    const startButton = modal.querySelector('.modal-card-foot .button');
+
+    // Fx to open the modal
+    function openModal() {
+        modal.classList.add('is-active');
+    }
+
+    // Fx to close the modal
+    function closeModal() {
+        modal.classList.remove('is-active');
+    }
+
+    // Event listener for the button to open the modal
+    modalButton.addEventListener('click', openModal);
+
+    // Event listener for the close button within the modal
+    closeButton.addEventListener('click', closeModal);
```

Comment:
Looks like a duplicate of line 36. Are both needed?

Reply:
Yes, both are needed. This is under the section for the user to be able to close the modal by clicking the 'Let's Get Started!' button, as stated in the comments under the section I made for it. For Line 36, that code is for the functionality of the 'x' button (I made a comment indicating a separate section for that one too).